### PR TITLE
[lit-next] LitElement removes the `lib` folder

### PR DESCRIPTION
**Pull request by sorvell**
Users will need to remove `lib/` from any import statements. For example:
* old: `import {UpdatingElement} from 'lit-element/lib/updating-element.js';`
* new: `import {UpdatingElement} from 'lit-element/updating-element.js';`

### CODE REVIEW

File: packages/lit-element/lit-element-polyfill.d.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+export * from './lit-element.js';
```
Comment (aomarks) [inline packages/lit-element/lit-element-polyfill.d.ts:14]: Should this file be here?

File: packages/lit-element/lit-element-polyfill.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import{UpdatingElement as t}from"./updating-element.js";export{UpdatingElement,defaultConverter,notEqual}from"./updating-element.js";import{render as e}from"lit-html";export*from"lit-html";export{CSSResult,css,supportsAdoptingStyleSheets,unsafeCSS}from"./css-tag.js";import{LitElement as o}from"./lit-element.js";export{LitElement}from"./lit-element.js";
```
Comment (aomarks) [inline packages/lit-element/lit-element-polyfill.js:1]: Should not be checked in

File: packages/lit-element/lit-element-polyfill.js.map [hunk 3]
```diff
@@ -0,0 +1 @@
...w.ShadyCSS!.styleElement(this);\n    }\n  };\n\n  (LitElement.prototype as any).adoptStyles = function (\n    this: LitElement,\n    styles: CSSResultArray\n  ) {\n    if (!(window.ShadowRoot && this.renderRoot instanceof window.ShadowRoot)) {\n      return;\n    }\n    window.ShadyCSS!.ScopingShim!.prepareAdoptedCssText(\n      styles.map((s) => (s as CSSResult).cssText),\n      this.localName\n    );\n  };\n}\n"],"names":["undefined","window","ShadyCSS","nativeShadow","LitElement","render","result","container","options","console","log","baseConnectedCallback","UpdatingElement","prototype","connectedCallback","call","this","hasUpdated","styleElement","adoptStyles","styles","ShadowRoot","renderRoot","ScopingShim","prepareAdoptedCssText","map","s","cssText","localName"],"mappings":";;;;;;;;;;;;;GAyBA,QAAwBA,IAApBC,OAAOC,WAA2BD,OAAOC,SAASC,aAAc,CAClEC,EAAWC,OAAS,CAClBC,EACAC,EACAC,KAEAC,QAAQC,IAAI,sCACZL,EAAOC,EAAQC,EAAWC,IAG5B,MAAMG,EAAwBC,EAAgBC,UAAUC,kBACxDV,EAAWS,UAAUC,kBAAoB,WACvCH,EAAsBI,KAAKC,MAGvBA,KAAKC,YACPhB,OAAOC,SAAUgB,aAAaF,OAIjCZ,EAAWS,UAAkBM,YAAc,SAE1CC,GAEMnB,OAAOoB,YAAcL,KAAKM,sBAAsBrB,OAAOoB,YAG7DpB,OAAOC,SAAUqB,YAAaC,sBAC5BJ,EAAOK,IAAKC,GAAOA,EAAgBC,SACnCX,KAAKY"}
```
Comment (aomarks) [inline packages/lit-element/lit-element-polyfill.js.map:1]: Should not be checked in

File: packages/lit-element/package.json [hunk 4]
```diff
@@ -14,9 +14,25 @@
       "development": "./development/lit-element.js",
       "default": "./lit-element.js"
     },
-    "./lib/": {
-      "development": "./development/lib/",
-      "default": "./lib/"
+    "./css-tag.js": {
```
Comment (aomarks) [inline packages/lit-element/package.json:17]: Also need to update the "files" section of this package.json or else they won't get published

Comment (sorvell) [reply packages/lit-element/package.json:17]: Updated.

File: packages/lit-element/lit-element-polyfill.js.map (hunk 3 above)
Comment (sorvell) [reply packages/lit-element/lit-element-polyfill.js.map]: Removed

File: packages/lit-element/lit-element-polyfill.js (hunk 2 above)
Comment (sorvell) [reply packages/lit-element/lit-element-polyfill.js]: Removed

File: packages/lit-element/lit-element-polyfill.d.ts (hunk 1 above)
Comment (sorvell) [reply packages/lit-element/lit-element-polyfill.d.ts]: Removed

**Review by aomarks — APPROVED** [review]

**Comment by sorvell** [comment]
Was handled via https://github.com/Polymer/lit-html/pull/1355

**Pull request closed without merging**